### ISSUE TITLE
Cannot show RateDiaLog 

### ISSUE THREAD

**Issue by hoangphuc1103** [Skyost/RateMyApp#147]
**Describe the bug**
I'm running the app on Android Simulator and am having trouble showing Rate Dialog - [Link](https://snipboard.io/HxituY.jpg)
**To Reproduce**
1. Go to Profile screen
2. Click on Rate the app
3. In the Debug Console and See error - https://snipboard.io/HxituY.jpg

**Expected behavior**
ShowRateDiaLog when click Rate the app option
**Screenshots**
1. [Link](https://snipboard.io/vS7Q9L.jpg)
2. [Link](https://snipboard.io/E5FrLY.jpg)
3. [Link](https://snipboard.io/V8TQwj.jpg)
4. [Link](https://snipboard.io/8mBiX5.jpg)
**Desktop (please complete the following information):**
 - OS: macOS Monterey 12.7.5
 - Browser: Chrome
 - Version 127.0.6533.89

**Smartphone (please complete the following information):**
 - Device: Android Emulator - Pixel_API_31
 - OS: 
 - Browser
 - Version 

**Additional context**

**Comment by Skyost** [comment]
It seems to be a problem with Play Core not showing the in-app review dialog. There are a lot of possible causes. Please see this answer : https://stackoverflow.com/a/65333746/3608831.

**Comment by chungxon** [comment]
Hey @Skyost, I think it might be related to #64

**Comment by Skyost** [comment]
Yes, it's possible. But sadly, as I stated on #64, there's really not much we can do...
